### PR TITLE
Backport Release 10.1.3 Changelog Updates

### PR DESCRIPTION
**Pull request by botanical**
**Summary:** Summary of changes

Addresses [CUMULUS-XX: Develop amazing new feature](https://bugs.earthdata.nasa.gov/browse/CUMULUS-XXX)

## Changes

* Detailed list or prose of changes
* ...

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

**Review by jennyhliu — APPROVED** [review]
👍

File: CHANGELOG.md [hunk 1]
```diff
@@ -6221,6 +6228,7 @@ Note: There was an issue publishing 1.12.0. Upgrade to 1.12.1.
 [v11.1.1]: https://github.com/nasa/cumulus/compare/v11.1.0...v11.1.1
 [v11.1.0]: https://github.com/nasa/cumulus/compare/v11.0.0...v11.1.0
 [v11.0.0]: https://github.com/nasa/cumulus/compare/v10.1.2...v11.0.0
```
Comment (npauzenga) [inline CHANGELOG.md:6230]: I think this needs to be `v10.1.3...v11.0.0`

**Review by npauzenga — CHANGES_REQUESTED** [review]
One note on the versions at the bottom of the changelog

File: CHANGELOG.md (hunk 1 above)
Comment (botanical) [reply CHANGELOG.md]: Ah nice catch!! Thanks for that, I've updated it https://github.com/nasa/cumulus/pull/3039/commits/aaea620f8fafc3fbe6f699e36dcd4a7c7b0e21e5

**Review by npauzenga — APPROVED** [review]
👍

**Pull request merged**